ext/oci8/oci8_statement.c

```diff
@@ -1717,11 +1708,7 @@ int php_oci_bind_array_by_name(php_oci_statement *statement, char *name, size_t
 
 	zvtmp = zend_string_init(name, name_len, 0);
 	zend_hash_update_ptr(statement->binds, zvtmp, bind);
-#if PHP_VERSION_ID < 70300
 	zend_string_release(zvtmp);
-#else
-	zend_string_release_ex(zvtmp, 0);
-#endif
```

Comment:
Should the new branch be retained?

Reply:
Yes, I missed those two when I did the other file.
Hopefully I fixed them all now. :)